src/test/regress/expected/cluster.out

```diff
@@ -494,11 +498,18 @@ SELECT * FROM clstr_expression WHERE upper(b) = 'PREFIX3';
 (1 row)
 
 EXPLAIN (COSTS OFF) SELECT * FROM clstr_expression WHERE -a = -3 ORDER BY -a, b;
-                          QUERY PLAN                           
----------------------------------------------------------------
- Index Scan using clstr_expression_minus_a on clstr_expression
-   Index Cond: ((- a) = '-3'::integer)
-(2 rows)
+                           QUERY PLAN                            
+-----------------------------------------------------------------
+ Gather Motion 3:1  (slice1; segments: 3)
+   Merge Key: b
+   ->  Sort
+         Sort Key: b COLLATE "C"
+         ->  Bitmap Heap Scan on clstr_expression
+               Recheck Cond: ((- a) = '-3'::integer)
+               ->  Bitmap Index Scan on clstr_expression_minus_a
+                     Index Cond: ((- a) = '-3'::integer)
```

Comment:
I think we should also do an `analyze` before these tests so that the plan is closer to what's in postgres.